crates/macro/src/html_tree/html_component.rs

```diff
@@ -123,10 +123,17 @@ impl ToTokens for HtmlComponent {
         };
 
         let set_children = if !children.is_empty() {
+            let i = (0..children.len())
+                .map(|x| Index::from(x))
+                .collect::<Vec<_>>();
             quote! {
-                .children(::yew::html::ChildrenRenderer::new(
-                    vec![#(#children.into(),)*]
-                ))
+                .children(::yew::html::ChildrenRenderer::new({
+                    let mut v = Vec::new();
+                    let comps = (#(#children,)*);
```

Comment:
I like this trick 👍

Reply:
I am reworking this as we speak. I really wanted #786 merged and now I will fix everything up to work with it.

Reply:
I will try pushing a working version today. These fixes, I think, will finally unblock my work.